Automattic/wp-calypso
# Reader: Adjust Colours

#### Changes proposed in this Pull Request

Intends to change the username and blog name in the reader to `color-primary` 

#### Testing instructions

Ensure that the colour change is now as expected when opening the Reader and clicking open a post. 

**Before:**

![hfdggfhdfgdh](https://user-images.githubusercontent.com/43215253/50237681-6cc00c00-03b5-11e9-8310-7e199e2b3dfd.png)

**After:**

![gsfdsfgdfsdgfgds](https://user-images.githubusercontent.com/43215253/50237698-777aa100-03b5-11e9-954c-caa0a26b51b5.png)

Fixes #29624
